topo_processor/util/transfer_collection.py

```diff
@@ -9,19 +9,34 @@
 
 async def transfer_collection(collection: Collection, target: str):
     stac_collection = collection.create_stac()
+    stac_collection.set_self_href("./collection.json")
```

Comment:
I would recommend setting this in the collection.create_stac() function. What do you think?